server/vulnerabilities/cpe_translations.json

```diff
@@ -0,0 +1,33 @@
+[
```

Comment:
Q: Should match have `platform` (`darwin`, `linux`, `windows`)?

E.g. `chrome_extensions` and `python_packages` as `source` are present in the three OSs.

Reply:
match only has access to fields present on `fleet.Software`. We do not have access to the host/platform, at least not without making significant changes to `CPEFromSoftware`. If there are differences between the software on the different platforms, then you can either uses a regex pattern for the name or just add multiple values for the name

```
    "match": {
      "name": ["name_on_windows", "name_on_linux"],
      "source": ["python_packges"]
    },
```

Reply:
Makes sense. In any case we could add new `"match"` fields later on.